lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -378,11 +378,11 @@
     description:
       - 'I(absent) - A container matching the specified name will be stopped and removed. Use force_kill to kill the container
          rather than stopping it. Use keep_volumes to retain volumes associated with the removed container.'
-      - 'I(present) - Asserts the existence of a container matching the name and any provided configuration parameters. If no
-        container matches the name, a container will be created. If a container matches the name but the provided configuration
-        does not match, the container will be updated, if it can be. If it cannot be updated, it will be removed and re-created
+      - 'I(present) - Asserts the existence of a docker image matching the name and any provided configuration parameters. If no
```

Comment:
I think this makes the docs more confusing. The module asserts that an image exists matching the `image:` option provided, and pulls that if it does not exist. saying "image matching the name" could cause users to conflate the `name:` parameter and the name provided to the `image:` parameter.